Mu2eG4/src/ConstructMaterials.cc

```diff
@@ -263,11 +263,25 @@ namespace mu2e {
       Polyethylene096->AddMaterial( findMaterialOrThrow("G4_C"), 0.86);
     }
 
+    mat = uniqueMaterialOrThrow( "Polyethylene094");
+    {
+      G4Material* Polyethylene094 = new G4Material( mat.name, 0.94*CLHEP::g/CLHEP::cm3, 2);
+      Polyethylene094->AddMaterial( findMaterialOrThrow("G4_H"), 0.14);
+      Polyethylene094->AddMaterial( findMaterialOrThrow("G4_C"), 0.86);
+    }
+
     mat = uniqueMaterialOrThrow( "Polyethylene0935");
     {
-      G4Material* Polyethylene096 = new G4Material( mat.name, 0.935*CLHEP::g/CLHEP::cm3, 2);
```

Comment:
Was Polyethylene096 having its definition overwritten before?  Do we know how long that's been happening, if so?  There will certainly be some difference due to the different density.  Hopefully small.

Reply:
This is just a change of the variable name, which only exists in the scope of the bracketed region. The name in the `uniqueMaterialOrThrow([name])` is the important naming part. I was just trying to improve the clarity by correcting the variable naming here.

Reply:
Thanks to Michael and Rob for pointing out the scoping at play.  Comment retracted.